### DISCUSSION TITLE
消除壓力的方式

### DISCUSSION THREAD

**Discussion by Taoyi-Chiang** [1101-GS4719-C/Python-Community-GS4719-C#2127] [General]
如果是以16週來計算的話，這學期也快過完四分之一了，工作量應該慢慢的越來越重，而且也讓人越來越覺得感到壓力。但其實我覺得消除壓力這件事情沒有這麼樣的困難，只是常常會忽略這些簡單的方式。
首先我覺得最有效的方式還是靜下心來，這種方式被廣泛的認為對於舒緩大腦的思考有很大的幫助，而我們的煩惱常常都是因為大腦毫無節制的思考造成的。這樣子說起來的話，我覺得一些古代僧人的修行方式，其實也有他自己的道理。
但有時候很難靜下心來，這時我會去室外走走。就我自己的工作經驗來說，其實每天只要能專注30分鐘就可以了，其他的時間如果沒有辦法專注，那麼繼續坐在電腦前也不會有任何效果。我感覺這時只要離開工作空間，通常再回來工作的時候都會很有效果。而且這時候不管是看中大湖的鵝還是去華泰來來往往的人們，都可以轉移佔據大腦認知空間的能力，進一步釋放壓力。我覺得這就有點像是刪除背景運作的程式來釋放記憶體空間，柴嘉尼效應的相關研究指出，如果一件事情沒有做完，會一直去想，我覺得這就是差不多的道理吧？
最後有另外一個我自己也很喜歡的方法，那就是找朋友聊天。有一種說法是，消除壓力的另外一個方式就是將困擾說出來。法國人最喜歡做的事情之一就是抱怨，我覺得這也是法國人總是讓人感覺到鬆弛感的原因。我想這種方式就像是心理學上說如果問題無法解決，那麼就是以情緒為導向為應對。其實我自己的感覺也確實是這樣，畢竟我們很多時候不能夠改變面對的狀況，像是天氣就不能改變，所以改變自己的情緒反而是最簡單的事情。
分享了一些我自己消除壓力的方式，當然保持運動也是很重要的，畢竟運動可以分泌多巴胺之類的物質，這樣就可以保持愉快的心情，當然也就不會感到那麼有壓力了。不知道還有沒有哪些消除壓力的方式呢？

**Comment by slps101023** [comment]
我覺得消除壓力的方法有很多種。我舉我自己為例，我參加系羽利用課餘時間和一群喜歡打羽球的人一起練球，切磋球技。雖然打完球之後很累，但不但可以認識不同年級的學長姐，也可以從他們口中聽到不同於同年齡的訊息，不只可以消除壓力，同時也可以學到很多課業外的知識，一舉兩得。

**Comment by NiCL100** [comment]
我覺得到聽松台眺望遠方頗能舒緩內心的壓力，看著藍天白雲以及高低參差、色彩斑斕的建築物，彷彿心胸也開闊了，能暫時忘記造成壓力的事物。感覺出門逛逛，將自己置於喧囂繁華中，亦能暫時放下壓力，但這在學校附近似乎無法實現。我也喜歡到圖書館走走，走在排滿書籍的書架中，會感到寧靜與祥和。我也十分認同文中提及的「抱怨」，這真的十分有效！

**Comment by Mminniehuang** [comment]
我覺得如果喜歡貓貓狗狗的話，可以去找找校貓校狗等等喜歡的小動物。以前高中讀書讀到壓力很大的時候，我們都會結伴去找校貓校狗，光是坐在牠們旁邊什麼都不做，身心靈就可以得到很大的滿足！
我記得之前某一堂課的老師在談到壓力這個主題的時候，也有分享她過去任教的某所國外大學舉辦了一個紓壓活動，參與的主角就是狗狗志工和壓力大的師生。
我覺得這是除了運動或是找人傾訴之外，很溫馨也很可愛的紓壓方式！

**Comment by Chattenoire** [comment]
我個人是透過閱讀自己喜歡的書來消解壓力，雖然別人都說這些書看起來根本就是增加壓力，但是我就喜歡讀 :3
至於到底讀什麼呢，大概像是人格權法、傅柯的性史、解殖相關的內容...etc

**Reply by kzx-creat** [reply]
想問一下，你喜歡的書是跟你的科系領域相關的嗎? 我上大學之後，原本當興趣會看的書幾乎都不太在閒暇時間看，因為跟科系所學太相關，反而有增加壓力的感覺。

**Comment by forestdew507** [comment]
推跑步！只要沒下雨可以免費使用的操場實在太方便了:) 可以邊跑邊放空或邊聽 podcast 都很讚XD

**Comment by Elaine4488** [comment]
我自己是透過看球賽的方式來解壓 ! 越緊張刺激的比賽，我越愛 ! 雖然有時候支持的球隊沒有贏會稍微難過一下下就是了

**Reply by Jing-Asteroid** [reply]
這樣不會更沮喪嗎?

**Reply by Elaine4488** [reply]
老實說有時候真的會影響心情，但我還是很享受投入比賽的過程。我覺得壓力的來源是因為太在意別人的看法，腦袋總是東想西想。這時候就能暫時放下，少一點思考的壓力。

**Comment by 112121py** [comment]
可以做一些自己喜歡的事情，或是讓自己能放空的東西，像樓上的跑步就蠻不錯的。我自己的話會吃東西、看喜歡的影集、或是就很單純的踩地雷放空也行xd

**Comment by Jing-Asteroid** [comment]
個人推薦暴力解壓法，去運動(最好是打球)，把壓力發洩在球上挺管用的，腦內啡也會幫助我們心情好，何樂而不為。
要不然就是騎車出去晃晃，看看大自然就會發現自己多麼渺小，就會覺得我們人類的壓力也不過爾爾，就釋然了。

**Comment by 113607514** [comment]
我消除自己壓力的方式是運動，不僅只是健身，打籃球也能讓我釋放壓力，也讓我的身體更健康強壯

**Comment by CeBru13** [comment]
我會看喜歡的小說或是在房間唱歌，不然就去找朋友打球聊屁話，真不行就去睡大覺，睡到自然醒，等你醒了壓力都沒了

**Comment by zhilin93517** [comment]
我也喜歡用睡覺的方式舒緩壓力，套一句古人常說的話:「睡覺治百病」
有時候甚至睡一覺起來更有想法做事更有效率。

**Comment by Denny3908** [comment]
我通常是唱歌、運動、找平常見不到的朋友出來。
不過如果壓力來源是工作、課業的話，個人經驗最好的方法就是用最高效率搞定工作，消除壓力源，避免所謂的「解壓活動」變成拖延造成心理壓力更大。

**Comment by lai1008** [comment]
我覺得消除壓力有各式各樣的方法。像是我就會選擇參加系羽和一群喜歡打羽球的人一起練球。雖然打完球之後很累，但不但可以認識不同年級的學長姐還可以消除壓力，同時也可以學到很多課業外的知識，一舉兩得。

**Comment by Jinyuu17** [comment]
1. 運動：進行有氧運動如跑步、瑜伽、健走能釋放腦內啡，提升情緒，有效減少壓力。
2. 深呼吸：專注於呼吸的節奏，緩慢地吸氣和吐氣，能幫助身體放鬆，立即減輕緊張感。
3. 正念冥想：通過專注當下、觀察自己的情緒和想法，來減少對未來或過去的焦慮，這種方法有助於長期減壓。
4. 與人交談：和朋友或家人分享你的感受，獲得情感支持，能有效減輕壓力和焦慮。
5. 放鬆的嗜好：聽音樂、看書、繪畫或從事園藝等活動，有助於轉移注意力，讓身心放鬆。
6. 睡眠充足：保證每晚的睡眠時間與質量，充足的休息能夠恢復精力，減少壓力對身體的影響。
7. 時間管理：有效的計劃和安排日常任務，避免過度繁忙和壓力堆積。

**Comment by lai1008** [comment]
超推跑步！只要沒下雨可以免費使用的操場實在太方便了，而且人也不多跑起來很舒服

**Comment by DarsooK** [comment]
透過時間管理來減少壓力的產生是一個不錯的方式，設立規律的讀書、工作及休息時間，讓自己覺得過得充實，也許就不會陷入焦頭爛額的情境。

**Comment by Tchang323** [comment]
消除壓力有非常多方法，我個人最推薦運動，不一定要跑步或式健身，可以是任何你喜歡的運動。運動能促進體內釋放內啡肽，一種讓人感到愉悅和放鬆的化學物質。幫助分散注意力，專注於動作與呼吸，從而減少焦慮。

**Comment by christaaah** [comment]
生活中確實會有很多壓力，最困擾我的除了學業壓力外就是社交壓力，特別到了大學後和不熟悉的人社交的場合越來越多，壓力就變得更大了。遇到這種時候，我通常都會自己一個人獨處，聽音樂、看影片、或是跟直播，踩地雷也是一種很解壓的方法。對我來說，只要暫時將自己的思緒從會讓自己感到壓力的地方抽離，壓力就不會那麼大，之後在處理時也能更冷靜。

**Comment by ShuliANN** [comment]
我覺得做一些自己喜歡的事情，不是逼迫自己去做碩會感到有壓力的，就是最好消除壓力的方式，像是我最喜歡的就是聽歌然後唱出來，或是去打打球也都很不錯！

**Comment by gigitilto** [comment]
我生活中的壓力來源主要都是社交壓力，當我撐不下去的時候，我會獨自一人待在沒有人在的空間聽著我喜歡的歌曲，偶爾逛逛討論區抒發一下當下的心情，或是看看一些養眼的圖片，這些都是我用來抒發壓力的方式。

**Comment by yuan3x2** [comment]
聽音樂、看小說、吃東西、睡覺，或做任何會讓我快樂的事，但我認為最快的減壓方法就是解決壓力的源頭，因此有時我會選擇一夜報肝解決工作。

**Comment by qiu0109** [comment]
我覺得與其坐在書桌前絞盡腦汁工作，不如去好好睡一覺或是看小說、追劇、洗熱水澡，讓心靈得到治癒說不定靈感就會突然出現，有了靈感事情就做得快，壓力自然也就變小了。

**Comment by lai1008** [comment]
我覺得做一些自己喜歡的事情就是最好消除壓力的方式，像是我最喜歡的就是聽歌然後唱出來，或是去打打球也都很不錯！

**Comment by wcyeee** [comment]
我消除壓力的方法是和家人或朋友說話！我會把所有造成我壓力的事情說給他們聽，告訴他們我為什麼會感到壓力，有些事情反而說不出什麼確切的感到壓力原因，這種事情通常講出來之後壓力就會減輕超級多了！如果是沒辦法消除的，想到有人也知道你的壓力，和你一起承受的話也會感覺好了許多！
